src/GeositeFramework/js/Plugin.js

```diff
@@ -32,7 +32,40 @@ require(['use!Geosite',
                 pluginName = model.get('pluginSrcFolder'),
                 $uiContainer = model.get('$uiContainer'),
                 $legendContainer = model.get('$legendContainer'),
-                logger = new Logger(pluginName);
+                logger = new Logger(pluginName),
+                resizers = {
+                    adjustBy: function(val) {
+                        var currentWidth = parseFloat($uiContainer.css("width"));
+                        var d = $.Deferred();
+                        d.then(function() {
+                            $uiContainer.css({ width: currentWidth + val });
+                        });
+                        return d.resolve();
+                    },
+                    setWidth: function(val) {
+                        var d = $.Deferred(),
+                            setter = function(w) {
+                                return function() {
+                                    $uiContainer.css({ width: "" });
+                                    $uiContainer.removeClass(function() {
+                                        var classes = this.className.split(" ");
+                                        return _.reject(classes, function(c) {
+                                            return /sidebar-width-.*/.test(c);
+                                        });
+                                    });
+                                    $uiContainer.addClass("sidebar-width-" + w);
+                                };
+                            };
```

Comment:
We can simplify this a bit since we're supporting a very limited number of sizes. Ex:

```
$uiContainer.css({ width: 'auto' })
    .removeClass('sidebar-width-small')
    .removeClass('sidebar-width-medium')
    .removeClass('sidebar-width-large')
    .addClass('sidebar-width-' + w);
```